src/software/ai/hl/stp/play/stop_play_test.cpp

```diff
@@ -12,11 +14,12 @@ class StopPlayTest : public SimulatedPlayTestFixture
 {
```

Comment:
You could pull some of the common setup functions into the SetUp of this fixture

```
 void SetUp() override
 {
    setFriendlyGoalie(0);
    addEnemyRobots(TestUtil::createStationaryRobotStatesWithId(
        {Point(1, 0), Point(1, 2.5), Point(1, -2.5), field().enemyGoalCenter(),
         field().enemyDefenseArea().negXNegYCorner(),
         field().enemyDefenseArea().negXPosYCorner()}));
    setEnemyGoalie(0);
    setAIPlay(TYPENAME(StopPlay));
    setRefereeCommand(RefereeCommand::STOP, RefereeCommand::STOP);
  }
```

This function will get called before every test

Reply:
Good idea! Fixed